### DISCUSSION TITLE
v0.45.1-20231211.195158

### DISCUSSION THREAD

**Discussion by qvacua** [qvacua/vimr#1032] [General]
* VimR now uses the release Neovim version 0.9.4 and is compatible with external Neovim binary version 0.9.x.
* Minium macOS requirement is now Monterey (v12).
* GH-1026: Bugfix: Scroll direction system setting is not respected when trackpad-scrolling.
* GH-1022: Bugfix: Custom Neovim binary setting is not stored correctly and the alert for incompatible version does not show up.
* Dependencies updates:
  - sparkle-project/Sparkle@2.5.1
  - Removed: Kitura/BlueSocket

<hr /><em>This discussion was created from the release <a href='https://github.com/qvacua/vimr/releases/tag/v0.45.1-20231211.195158'>v0.45.1-20231211.195158</a>.</em>

**Comment by kuntau** [comment]
Hi @qvacua,

May I know what's the reason for MacOS version bump? Asking because I'm still running `Big Sur`. I noticed `neovide` also recently bump minimum MacOS requirements.
